garage/misc/logger/__init__.py

```diff
@@ -1,11 +1,11 @@
 """Global logger module.
 
 This module instantiates a global logger according to configuration.
-Use LOG_TENSORBOARD=True in config.py to enable Tensorboard.
+Use GARAGE_LOG_TENSORBOARD=True in config.py to enable Tensorboard.
```

Comment:
"Set the environment variable GARAGE_LOG_TENSORBOARD=True to enable Tensorboard"